pkg/sharings/sharings.go

```diff
@@ -157,7 +157,13 @@ func findSharingRecipient(db couchdb.Database, sharingID, clientID string) (*Sha
 func addTrigger(instance *instance.Instance, rule permissions.Rule, sharingID string) error {
 	scheduler := instance.JobsScheduler()
 
-	eventArgs := rule.Type + ":UPDATED:" + strings.Join(rule.Values, ",")
+	var eventArgs string
+	if rule.Selector != "" {
+		eventArgs = rule.Type + ":CREATED,UPDATED,DELETED:" + rule.Values[0] + ":" + rule.Selector
```

Comment:
The syntax is designed so there can also be a selector by field = one of multiple value, not sure if you removed that on purpose.